hudi-utilities/src/test/java/org/apache/hudi/utilities/deltastreamer/TestHoodieDeltaStreamer.java

```diff
@@ -2827,7 +2828,7 @@ public void testAutoGenerateRecordKeys() throws Exception {
     deltaStreamer.sync();
     assertRecordCount(parquetRecordsCount, tableBasePath, sqlContext);
     // validate that auto record keys are enabled.
-    HoodieTableMetaClient metaClient = HoodieTableMetaClient.builder().setBasePath(tableBasePath).setConf(jsc.hadoopConfiguration()).build();
+    HoodieTableMetaClient metaClient = HoodieTableMetaClient.builder().setBasePath(tableBasePath).setConf(new HadoopStorageConfiguration(jsc.hadoopConfiguration())).build();
```

Comment:
Can't we use the `HoodieTestUtils.getDefaultStorageConf`?

Reply:
sure